Front-End/challenge1/script.js

```diff
@@ -1 +1,28 @@
 // Modify this file only
+'use strict'
+
+let counter = 0;
+const increase = document.getElementById('increase');
+const decrease = document.getElementById('decrease');
+
+
+
+const Increase = () => {
```

Comment:
please stick to the camelcase naming convention

Reply:
Fixed